Google AI.js

```diff
@@ -0,0 +1,188 @@
+	"translatorID": "5f1c4a3b-b7cf-4170-a896-e4d82c0621c9",
+	"label": "Google AI",
+	"creator": "Guy Aglionby",
+	"target": "^https://ai\\.google/research/(pubs|people)",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2018-09-09 17:55:01"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2018 Guy Aglionby
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero.  If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+var bibtex2zoteroTypeMap = {
+	"inproceedings": "conferencePaper",
+	"conference"   : "conferencePaper",
+	"article"      : "journalArticle",
+	"techreport"   : "report",
+	"patent"       : "patent"
+};
+
+function detectWeb(doc, url) {
+	if (url.includes('/pubs/pub')) {
+		let bibtex = extractBibtex(doc);
+		let doctype = bibtex.split('{')[0].replace('@', '');
+		return bibtex2zoteroTypeMap[doctype];
+	} else if (getSearchResults(doc, true)) {
+		return 'multiple';
+	}
+}
+
+function doWeb(doc, url) {
+	let detectType = detectWeb(doc, url);
+	if (detectType === 'multiple') {
+		// Test cases: (defer: true is broken so cannot currently use automated tests)
+		// https://ai.google/research/people/105197
+		// https://ai.google/research/pubs/
+		Zotero.selectItems(getSearchResults(doc), function (selected) {
+			if (!selected) {
+				return true;
+			}
+			ZU.processDocuments(Object.keys(selected), scrape);
+		});
+	} else if (detectType) {
+		scrape(doc, url);
+	}
+}
+
+function scrape(doc, url) { 
+	let bibtex = extractBibtex(doc);
+	let translator = Zotero.loadTranslator("import");
+	translator.setTranslator("9cb70025-a888-4a29-a210-93ec52da40d4");
+	translator.setString(bibtex);
+	translator.setHandler("itemDone", function (obj, item) {
+		let downloadElement = ZU.xpath(doc, '//a[span[contains(@class, "icon--download")]]');
+		if (downloadElement.length) {
+			let downloadUrl = downloadElement[0].href;
+			if (downloadUrl.endsWith('.pdf')) {
+				item.attachments.push({
+					url: downloadUrl,
+					title: 'Full Text PDF',
+					mimeType: 'application/pdf'
+				});
+			}
+		}
+		delete item.itemID;
+		item.complete();
+	});
+	translator.translate();	
+}
+
+function extractBibtex(doc) { 
+	let bibtexElement = ZU.xpath(doc, '//a[span[contains(@class, "icon--copy")]]')[0];
+	let escapedBibtex = bibtexElement.getAttribute('ng-click')
```

Comment:
any reason you're not just using `ZU.xpathText(doc, '//a[span[contains(@class, "icon--copy")]]/@ng-click')`? or the `attr` function with a CSS selector? Same above for getting the PDF